src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -1863,13 +1862,14 @@
   "modifier.tconstruct.leaping.description": "兔子可以跳得很高，那么这个应该也能让你跳得很高",
   "modifier.tconstruct.pockets": "行囊",
   "modifier.tconstruct.pockets.flavor": "真是奇妙的发明！",
-  "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，潜行时按下“护腿交互”键就可以存取",
+  "modifier.tconstruct.pockets.description": "这护腿是有口袋的，能让你存储一些额外的物品，潜行时按下“护腿交互”键就可以存取 The pants have pockets, allowing you to store additional items which can be accessed by shift plus interacting",
   "modifier.tconstruct.shield_strap": "副手套环",
   "modifier.tconstruct.shield_strap.flavor": "保护与攻击的理想之选",
   "modifier.tconstruct.shield_strap.description": "按下“护腿交互”键能够交换副手物品",
   "modifier.tconstruct.tool_belt": "工具腰带",
   "modifier.tconstruct.tool_belt.flavor": "恨不得长出六只手么？",
-  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏",
+  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏 Interacting with the leggings whle holding control allows swapping the hotbar",
+  "recipe.tconstruct.modifier.tool_belt": "Tool belt recipes must be applied in order",
```

```suggestion
  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏",
  "recipe.tconstruct.modifier.tool_belt": "Tool belt recipes must be applied in order",
```

```suggestion
  "modifier.tconstruct.tool_belt.description": "按下护腿交互键能够交换快捷栏",
  "recipe.tconstruct.modifier.tool_belt": "“工具腰带”需按顺序施加",
```
*Can't find where the latter key would appear*